packages/luvi/build.sh

```diff
@@ -17,36 +17,29 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 	-DWithPCRE2=On
 	-DWithSharedPCRE2=On
 	-DWithLPEG=On
+	-DLIBLUV_INCLUDE_DIR=${TERMUX_PKG_SRCDIR}/deps/luv/src
+	-DLIBLUV_LIBRARIES=${TERMUX_PREFIX}/lib/libluv.so
+	-DLUAJIT_INCLUDE_DIR=${TERMUX_PREFIX}/include/luajit-2.1
+	-DLUAJIT_LIBRARIES=${TERMUX_PREFIX}/lib/libluajit.so
+	-DLIBUV_INCLUDE_DIR=${TERMUX_PREFIX}/include
+	-DLIBUV_LIBRARIES=${TERMUX_PREFIX}/lib/libuv.so
+	-DOPENSSL_INCLUDE_DIR=${TERMUX_PREFIX}/include
+	-DOPENSSL_LIBRARIES=${TERMUX_PREFIX}/lib
+	-DPCRE2_INCLUDE_DIR=${TERMUX_PREFIX}/include
+	-DPCRE2_LIBRARIES=${TERMUX_PREFIX}/lib
 "
 
 termux_step_post_get_source() {
 	git checkout "${_commit}"
 }
 
 termux_step_pre_configure() {
-	declare include=${TERMUX_PREFIX}/include lib=${TERMUX_PREFIX}/lib
-	TERMUX_PKG_EXTRA_CONFIGURE_ARGS+="
-		-DLIBLUV_INCLUDE_DIR=${TERMUX_PKG_SRCDIR}/deps/luv/src
-		-DLIBLUV_LIBRARIES=$lib/libluv.so
-		-DLUAJIT_INCLUDE_DIR=$include/luajit-2.1
-		-DLUAJIT_LIBRARIES=$lib/libluajit.so
-		-DLIBUV_INCLUDE_DIR=$include
-		-DLIBUV_LIBRARIES=$lib/libuv.so
-		-DOPENSSL_INCLUDE_DIR=$include
-		-DOPENSSL_LIBRARIES=$lib
-		-DPCRE2_INCLUDE_DIR=$include
-		-DPCRE2_LIBRARIES=$lib
-	"
```

Comment:
Seems like all the options are duplicated here, I don't think that should be necessary. Please correct me if I am missing something

Reply:
I don't get the meaning of "duplicated" here, could you rephrase it?

Reply:
Oh, sorry, I think I was fooled by my browser, I thought these lines were added but they are removed. Ignore this :)